### PR TITLE
Feature/next/articles

### PR DESCRIPTION
**Pull request by dw-buildit**
Adds articles to `next` including some simple metadata, list outputs, images etc

### CODE REVIEW

File: layouts/partials/articles/hero.njk [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<h1>{{ title }}</h1>
+<p>{{ description }}</p>
+<h4>{{ humanDate }}</h4>
```
[the remark this answers is not in the document]
Comment (dw-buildit) [reply layouts/partials/articles/hero.njk]: Will look into

File: lib/metalsmith-path-noindex.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+const path = require("path");
```
[the remark this answers is not in the document]
Comment (dw-buildit) [reply lib/metalsmith-path-noindex.js]: For reference, this is a direct copy of the original metalsmith-path plugin (it doesn't do anything special!) here: https://github.com/greduan/metalsmith-path/blob/master/lib/index.js

The issue we have is that it we would have to use it many times to strip 'index.html', 'index.md', 'index.html.md', '.njk' and combinations thereof. The directoryIndex property doesn't take an array.

The alternative is running the path plugin late in the metalsmith order - but then we lose access to the variables when we're rendering njk templates or frontmatter.

File: lib/sorts/named-index-sort.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+module.exports = indexName => (a, b) => {
```
[the remark this answers is not in the document]
Comment (dw-buildit) [reply lib/sorts/named-index-sort.js]: will do

File: lib/metalsmith-path-noindex.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (james-nash) [reply lib/metalsmith-path-noindex.js]: Ah, OK. I wasn't aware of that.

Alrighty then. Carry on. As you were.

**Pull request merged**